lib/components/textEditor/richText.js

```diff
@@ -115,7 +116,12 @@ class RichText extends Component {
             ref={(element) => {
               this.editor = element;
             }}
-            placeholder={placeholder}
+            placeholder={
+              // eslint-disable-next-line react/destructuring-assignment
+              this.initialEditorContent === stateToHTML(this.state.editorState.getCurrentContent())
```

Comment:
Nit: You already have a `editorState` variable we can use

Reply:
It seems strange that we need to fix the placeholder this way. I would expect the library we use to know when to display the placeholder automatically. Do we know why this is happening?